examples/demoTrades/README.md

```diff
@@ -2,3 +2,5 @@
  - Example schema for trade list
  - This profile generates data that does have dependencies between some columns
    (settlement date = trade date + 5 working days, trade price ranges depend on stock code)
+
+Since this profile uses an external file it is not supported by the datahelix playground.
```

Comment:
It would be good if all of the examples that can't be run in the playground had the same phrasing - I prefer the "This profile is not currently supported by the datahelix playground as it requires an external file." version